include/astarte_device_sdk/formatter.hpp

```diff
@@ -0,0 +1,456 @@
+// (C) Copyright 2025, SECO Mind Srl
```

Comment:
Why didn't you separate the header form the source implementation in two different file?
- formatter.hpp
- formatter.cpp